torchaudio/csrc/register.cpp

```diff
@@ -77,5 +77,17 @@ TORCH_LIBRARY(torchaudio, m) {
   m.def(
       "torchaudio::sox_effects_apply_effects_file",
       &torchaudio::sox_effects::apply_effects_file);
+
+  //////////////////////////////////////////////////////////////////////////////
+  // transducer.cpp
+  //////////////////////////////////////////////////////////////////////////////
+  m.def("rnnt_loss(Tensor acts,"
```

Comment:
We need a guard for excluding the prototype from the binary for the release process.
We need to decide wether we want to build prototype by default, and define a way to toggle it, (which is starting from an environment variable, then passing/parsing that in CMake, then passing it as compiler definition, which will be tested with `#ifdef` macro) then modify the CI configuration so that nightly build contains prototype, but proper release won't.

Reply:
Also to decouple the different bindings. We should use `TORCH_LIBRARY_FRAGMENT` and register `rnnt` inside of `transducer.cpp`